website/src/data/users.tsx

```diff
@@ -95,12 +95,12 @@ const Users: User[] = [
     tags: ['opensource', 'personal'],
   },
   {
-    title: 'Algolia Docsearch',
+    title: 'Algolia DocSearch',
     description:
       'The best search experience for docs, integrates in minutes, for free',
     preview: require('./showcase/algolia.png'),
     website: 'https://docsearch.algolia.com/',
-    source: 'https://github.com/algolia/docsearch-website',
+    source: 'https://github.com/algolia/docsearch',
```

Comment:
```suggestion
    source: 
      'https://github.com/algolia/docsearch/tree/next/packages/website',
```

You would prefer `next`, right?

Reply:
It's already the default branch and I'll do the renames soon so I guess we can keep the scope large to the repo, wdyt?

Reply:
Ok make sense! I'll do it tomorrow and ping you here once done :D

Reply:
Great, thanks!

Reply:
```suggestion
    source:
      'https://github.com/algolia/docsearch/tree/main/packages/website',
```

Reply:
All good!